lua/nvim-treesitter/install.lua

```diff
@@ -67,11 +68,15 @@ local function get_parser_install_info(lang, validate)
   return install_info
 end
 
+local function get_package_path(...)
```

Comment:
this is used in `./scripts/write-lockfile.lua`, btw.

(but could just as well be inlined there, of course)